translations/base-nl.yaml

```diff
@@ -117,8 +117,8 @@ dialogs:
         title: Savegame geïmporteerd
         text: Je savegame is succesvol geïmporteerd.
     gameLoadFailure:
-        title: Het spel is kapot
-        text: "Het laden van je savegame is mislukt:"
+        title: Corrupte savegame
+        text: Het laden van je savegame is mislukt:
```

Comment:
The quotes should stay at line 121 otherwise you get the lint-check error.

Reply:
done